tensorboard/webapp/metrics/views/metrics_container.scss

```diff
@@ -16,12 +16,17 @@ limitations under the License.
 
 :host {
   display: flex;
-  height: 100%;
   flex-direction: column;
+  height: 100%;
+  justify-content: stretch;
   overflow: hidden;
 }
 
 .notice {
+  background-color: rgba(mat-color($mat-yellow, 200), 0.85);
+  border-bottom: 1px solid mat-color($mat-yellow, 500);
+  color: #000;
```

Comment:
The text color and background contrast is a bit high to me. Can we change it to "rgb(33,33,33)" as other black in dashboard?
perhaps `color: inherit;`

Reply:
Cannot inherit since I don't want colors to change in the dark mode.